packages/office-addin-debugging/src/start.ts

```diff
@@ -52,9 +61,32 @@ export async function isPackagerRunning(statusUrl: string): Promise<boolean> {
 export function parseAppType(text: string): AppType | undefined {
     switch (text) {
         case "desktop":
+        case "macos":
+        case "win32":
             return AppType.Desktop;
         case "web":
             return AppType.Web;
+        case "ios":
+        case "android":
+            throw new Error(`Platform type ${text} not currently supported for debugging`);
+        default:
+            return undefined;
+    }
+}
+
+export function parseAppPlatform(text: string): AppPlatform | undefined {
+    switch (text) {
+        case "desktop":
+            return process.platform === "win32" ? AppPlatform.Win32 : AppPlatform.MacOS;
```

Comment:
Use a switch. For "win32" and "darwin" return Platform.Win32 / MacOS. For the default case, throw new Error(`The current platform is not supported: ${process.platform}`);